src/main/jbake/content/documentation/bundles/models.md

```diff
@@ -179,92 +150,121 @@ Since Sling Models 1.2.0 there is another way of instantiating models. The OSGi
         // See the javadoc of the ModelFactory for which Exception can be expected here
     }
 
-In addition `ModelFactory` provides methods for checking whether a given class is a model at all (having the model annotation) or whether a class can be adapted from a given adaptable.
+In addition `ModelFactory` provides methods for checking whether a given class is a model at all (having the model annotation) or whether a class can be adapted from a given adaptable. 
 
 ## Usage in HTL
-[Sling Models Use Provider](/documentation/bundles/scripting/scripting-htl.html#java-use-provider-1) (internally uses the `ModelFactory` from above).
+Please see [Sling Models Use Provider](/documentation/bundles/scripting/scripting-htl.html#java-use-provider-1); internally it uses the `ModelFactory` from above.
 
-# Other Options
-## Names
-If the field or method name doesn't exactly match the property name, `@Named` can be used:
 
-	::java
-	@Model(adaptables=Resource.class)
-	public class MyModel {
-	 
-	    @Inject @Named("secondPropertyName")
-	    private String otherName;
-	} 
- 
+# Available injectors
+
+In the above cases just the `@ValueMapValue` annotation was used, but there other available injectors. For each injector there is a specialized annotation available. For the optional parameters see the next section.
+
+
+Injector Name  | Annotation          | Supported Optional Elements    | Description   | Array Support   | Parametrized Type Support
```

Comment:
The first column is now a label and no longer the name. The name should still be documented as it must be used with annotation `@Source`

Reply:
I added more rows to the table and included the name and the column "Applicable to".